### DISCUSSION TITLE
Metal rendering issues on 3802, Ventura 13.3

### DISCUSSION THREAD

**Discussion by ParaDoX1994** [dortania/OpenCore-Legacy-Patcher#1044] [Reports being investigated]
Following Applications error due to Metal rendering issues:

* Blender
* Parallels Desktop
  * [See work-around](https://forum.parallels.com/threads/opencore-legacy-graphic-driver-vs-parallels-desktop-revert-to-make-it-work.359469/#post-915059)

More applications directly invoking the Metal framework are likely affected. Downgrade to macOS 13.2.1 or older if these applications are critical to your workflow

---------

Blender error attached:
<img width="372" alt="228114231-87bb08d3-a59a-41bc-b388-5340a86224e9" src="https://user-images.githubusercontent.com/35791009/228909880-f6adb7bf-d146-4830-a0bc-e9faf1b11663.png">

**Comment by crystall1nedev** [comment]
Possibly related, issue with System Settings' sidebar icons not rendering.

**Comment by khronokernel** [comment]
https://github.com/dortania/OpenCore-Legacy-Patcher/commit/1ce6f792409faddec56bee9858e50bc2bae2363b
